documentation/docs/overview.md

```diff
@@ -1,35 +1,69 @@
 ---
-description: The wallet library is a stateful package with a standardized interface for developers to build applications involving IOTA value transactions.
+description: 'The iota.rs library is designed to simplify how you connect and interact
+with [nodes](https://wiki.iota.org/chrysalis-docs/node_software) in the IOTA network. '
 image: /img/overview/layered_overview.svg
 keywords:
+
 - layered overview
 - high level
 - low level
 - stronghold
 - value transactions
+- explanation
+
 ---
+
 # Overview
 
-To communicate with the IOTA network, you have to connect and interact with a [node](https://wiki.iota.org/chrysalis-docs/node_software), the task that the iota.rs library is designed to simplify. It will generate addresses for you, send blocks, sign and send transactions, and more.
+![IOTA Client Library Overview](/img/Banner/banner_client_lib_overview.png)
+
+The iota.rs library is designed to simplify how you connect and interact
+with [nodes](https://wiki.iota.org/introduction/develop/explanations/node_software) in the IOTA network. You can use it to generate
+addresses, send messages, sign and send transactions, and more.
 
-Beyond establishing the initial connection to a node, iota.rs has no state. Operations use only the data that you pass during a call and have no effect on your software beyond returning a value. You are in full control of the data flow in your application.
+Beyond establishing the initial connection to a node, iota.rs has no state. Operations use only the data you pass during
+a call and have no effect on your software beyond returning a value. You are in complete control of the data flow in
+your application.
 
-This stateless approach makes iota.rs easier for you to use and understand. But since you are in full control of the data management, you also fully responsible for it, which could feel overwhelming if you have to handle complex or sensitive data. If you plan on managing funds in your application, take a look at our [wallet.rs](https://wiki.iota.org/wallet.rs/welcome) library instead. It allows you to safely manage your user's funds, and it already includes our best security practices. It uses [stronghold.rs](https://wiki.iota.org/stronghold.rs/welcome) to store sensitive data and iota.rs to communicate with the IOTA network. Unlike iota.rs, it has a state.
+This stateless approach makes iota.rs easy to use and understand. But since you are in full control of data management,
+you are also fully responsible for it, which could feel overwhelming if you handle complex or sensitive data. If you
+plan on managing funds in your application, take a look at
+our [wallet.rs library](https://wiki.iota.org/wallet.rs/welcome) instead. It allows you to manage your user's funds
+safely, and it already includes our best security practices. It
+uses [stronghold.rs](https://wiki.iota.org/stronghold.rs/welcome) to store sensitive data and iota.rs to communicate
```

Comment:
wallet.rs can be used without stronghold and iota.rs can also be used together with stronghold now, so I think this point shouldn't be there anymore